.github/workflows/localstack-example.yml

```
@@ -0,0 +1,46 @@
+name: LocalStack example pipeline
```

Also, rename the file

```suggestion
name: LocalStack Module pipeline
```

---

Done